packages/core/webpack/dev-client.js

```diff
@@ -14,6 +14,7 @@ const commonLoaders = require(loaderPath);
 
 process.traceDeprecation = true;
 
+/** @type {import("webpack").Configuration} */
```

Comment:
Is this some sort of JSDoc import of the specific `type` we're exporting here? Cool!

Reply:
It's a JSDoc typing. I've been using these for months, Intellisense caughts them and tell me about allowed types and errors (if there's a jsconfig.json file in the project).